C/ChibiScheme/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+using BinaryBuilder
+
+name = "ChibiScheme"
+version = v"0.10.1" # NOT OFFICIAL
+
+# Collection of sources required to build NLopt
+sources = [
+    GitSource("https://github.com/ashinn/chibi-scheme.git",
+              "b0735b3ca70620face209c5066898e5e9e1fcf62"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/chibi-scheme
+mkdir build && cd build
```

Comment:
It provides the command `chibi-scheme` which, as far as I can tell, isn't built anywhere by this build system, not even for x86_64-linux-musl which is a pure native build, not a cross one.

Reply:
No, wait, [it's built](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=19606&view=logs&j=abd022a1-9f63-5341-d138-0709aa033c2f&t=ac164843-b50a-5bf6-a62e-0593feb78eb2&l=386) but then they magically expect it to be in the PATH?

Reply:
Anyway, this wouldn't work for all platforms because we're building for the target system, and CMake conveniently does _**not**_ allow you to specify a host toolchain and a target toolchain at the same time (contrary to autoconf with which you can set `CC_BUILD` and `CC` separately, which is what we do automatically in BinaryBuilder).

So the options are:

1. to do a native bootstrap and use that to build the package for the target,
2. or to avoid manually doing the bootstraps and just use the package from Alpine Linux to get the `chibi-scheme` command.